app/services/github_checks_verifier.rb

```diff
@@ -11,6 +11,9 @@ class GithubChecksVerifier < ApplicationService
   config_accessor :check_name, :workflow_name, :client, :repo, :ref
   config_accessor(:wait) { 30 } # set a default
   config_accessor(:check_regexp) { "" }
+  config_accessor(:allowed_conclusions){ "success,skipped" }
+
+  allowed_conclusions_array = allowed_conclusions.split(/\s*,\s*/)
```

Comment:
```suggestion
  config_accessor(:allowed_conclusions) { ["success", "skipped"] }
```
And you don't need to use the split and a new variable